pom.xml

```diff
@@ -320,6 +320,31 @@ under the License.
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-shade-plugin</artifactId>
             </plugin>
+			<plugin>
+				<groupId>org.codehaus.mojo</groupId>
+				<artifactId>flatten-maven-plugin</artifactId>
+				<version>1.1.0</version>
```

Comment:
Can we use 1.5.0? Its newer and also used by the spark bq connector https://github.com/GoogleCloudDataproc/spark-bigquery-connector/blob/881d2cca5a6dfe34cc8151a00916bbdcafff21fc/spark-bigquery-parent/pom.xml#L538